app/styleguide/Clickr.guide.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { SGPage, SGSection } from 'reapp-dev-tools';
+
+
+import Clickr from 'components/Clickr';
+
+export default class ClickrGuideComponent extends React.Component {
+    render() {
+        return (
+            <SGPage>
+
+                <SGSection title="Clickr - without properties">
+                    <Clickr />
+                </SGSection>
+
+                <SGSection title="Clickr - with value property">
+                    <Clickr value="Content for the component" />
```

Comment:
Actually I didn't use the Styleguide file because i can see that component works without making any changes in guide.js file. I cleaned it up anyway.